consumer/dragonboard/qualcomm-robotics-rb5/installation/linux-fastboot.md

```diff
@@ -0,0 +1 @@
```

#### **Step 5**: Recall location of all downloaded files

Recall location of all downloaded files from the downloads page, files will be different for Android and Linaro/Debian:

###### **Linaro/Debian**: Recall location of `boot` and `rootfs` downloaded from the downloads page
- You should have downloaded the `boot` file
- You should have downloaded ONE of rootfs` file (Either `Developer` or `Desktop - ALIP` version)

###### **AOSP**: Recall location of `boot.img.tar.xz`, `system.img.tar.xz`, `userdata.img.tar.xz`, `recovery.img.tar.xz`, `persist.img.tar.xz`, `cache.img.tar.xz`, downloaded from the downloads page
- All of these files should have been downloaded from the downloads page

#### **Step 6**: Unzip required files

##### For Debian
```shell
$ gunzip boot-*.img.gz
$ gunzip linaro-*.img.gz
```

##### For AOSP
```shell
$ tar -xvf boot.img.tar.xz
$ tar -xvf system.img.tar.xz
$ tar -xvf userdata.img.tar.xz
$ tar -xvf recovery.img.tar.xz
$ tar -xvf persist.img.tar.xz
$ tar -xvf cache.img.tar.xz
```

#### **Step 7**: Flash all images to the RB5

- Use host computer
- Use "Terminal" application
- Recall location of extracted(unzipped) files
- `cd` to the directory with your unzipped files
- From within extraction directory, execute the following commands:

###### **Linaro/Debian**:
```shell
# (Once again) Check to make sure fastboot device connected
$ sudo fastboot devices
# It will show similar to below if the device is connected successfully
de82318	fastboot

# cd to the directory the boot image and  were extracted
$ cd <extraction directory>

# Make sure you have properly unzipped the boot and rootfs downloads
$ sudo fastboot flash boot boot-linaro-buster-qrb5165-**BUILD#**.img
$ sudo fastboot flash rootfs linaro-buster-developer-qrb5165-**BUILD#**.img
```
**Note**: Replace **BUILD#** in the above commands with the file-specific date/build stamp.

###### **AOSP**:
```shell
# (Once again) Check to make sure fastboot device connected
$ sudo fastboot devices
# It will show similar to bellow if the device is connected successfully
de82318	fastboot

# cd to the directory with extracted images
$ cd <extraction directory>

# Make sure you have properly unzipped the downloads
$ sudo fastboot flash boot boot.img
$ sudo fastboot flash system system.img
$ sudo fastboot flash userdata userdata.img
$ sudo fastboot flash recovery recovery.img
$ sudo fastboot flash persist persist.img
$ sudo fastboot flash cache cache.img
```

#### **Step 8**: Reboot RB5

- Unplug power to RB5
- Ensure HDMI connection to monitor
- Ensure keyboard and/or mouse connection (Depending on your rootfs selection)
- Plug power back into RB5
- Wait for board to boot up
- Board will boot into either command line or desktop depending on rootfs

**Note:** For Linaro/Debian the **username** and **password** are both **“linaro”** when the login information is requested.

**Congratulations! You are now booting your newly installed OS directly from UFS on the RB5!**


```

Comment:
formatting needs to be fixed, weird `^M` characters appearing in-place of  regular `<enter>/<return>`